server/db/Seeder.js

```diff
@@ -1,15 +1,32 @@
 import pg from "pg"
 import path from "path"
-import LineReader from "line-reader"
+//import LineReader from "line-reader"
 import { fileURLToPath } from "url"
+import fs from 'fs'
 
 const pool = new pg.Pool({
-  connectionString: "postgres://postgres:password@localhost:5432/<yourDBnameHere>" })
+  connectionString: "postgres://postgres:password@localhost:5432/pets-app" });
 
 class Seeder {
   static async seed() {
-    // your seeder code here
+    let loadedData = fs.readFileSync('data.json');
+    let contents = JSON.parse(loadedData);
+    
+    //console.log(contents.data.petTypes[0]);
+    let petTypes = contents.data.petTypes;
+    try{
+      petTypes.forEach(async (petType) => {
+          let query = 'INSERT INTO pet_types (type, img_url, description) VALUES ($1, $2, $3) RETURNING id';
```

Comment:
not required but a suggestion to consider - this query will return the `id` of each new record, and in this instance for inserting, the `id` isn't used for anything! Here, we can omit the RETURNING clause